modules/minifact/tasks/powershell.ps1

```diff
@@ -0,0 +1,53 @@
+#!powershell.exe
+
+# Delegate to facter if available
+if (Get-Command facter -ErrorAction SilentlyContinue) {
+    facter --json
```

Comment:
I think at some point we'll want to unify finding Facter on Windows between this and the Ruby version. Not sure how yet.

Reply:
We can re-implement the same logic, but I don't see a way how we could share it.

Reply:
I think we can probably do away with the ruby one once we can mutliple implementations